### PR TITLE
Blog post for 1.11 release.

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

File: _posts/2021-08-09-CocoaPods-1.11.0-beta.markdown [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  spec.on_demand_resources = {
+    'level1_assets' => ['level1/**/*.png'],
+    'level2_assets' => ['level2/**/*.png']
+  }
+end
+```
+
+Each key specifies a tag and each value is a glob for the paths to include. 
+
+> Note: On demand resource paths specified within `on_demand_resources` do *not* require to be also specified within `resources` DSL or `resource_bundle` DSL.
```
Comment (paulb777) [inline _posts/2021-08-09-CocoaPods-1.11.0-beta.markdown:41]: s/do *not* require/are *not* required/

File: _posts/2021-08-09-CocoaPods-1.11.0-beta.markdown [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+* `pre_integrate_hook` DSL: Similar to the `post_integrate_hook` DSL added in version [1.10](https://blog.cocoapods.org/CocoaPods-1.10.0-beta/), this new hook will be executed before the user's project is integrated with the generated Pods project. 
+
+### Xcode 13 Support
+
+A new version of the `Xcodeproj` gem has shipped that adds initial support for Xcode 13 projects. To update, run `gem update xcodeproj` or if you are using [Bundler](https://bundler.io/) then run `bundle update xcodeproj` instead.
+
+CocoaPods 1.11 by default includes the latest version of the `Xcodeproj` gem allowing it to work with Xcode 13 right out of the box.
+
+That's it!
+
```
Comment (paulb777) [inline _posts/2021-08-09-CocoaPods-1.11.0-beta.markdown:72]: Can a definitive statement or pointer to M1 support resolution be added?

**Review by paulb777 — APPROVED** [review]
Looks great!

File: _posts/2021-08-09-CocoaPods-1.11.0-beta.markdown (hunk 2 above)
Comment (dnkoutso) [reply _posts/2021-08-09-CocoaPods-1.11.0-beta.markdown:72]: good point, I do not have an M1 available but we could talk about downstream dependencies and perhaps the fact we are trying to add CI support for it?

Comment (dnkoutso) [reply _posts/2021-08-09-CocoaPods-1.11.0-beta.markdown:72]: what I meant to also say Is that I do not have definitive answers regarding M1 support to post them in the blog post...thats what "scares" me.

Comment (paulb777) [reply _posts/2021-08-09-CocoaPods-1.11.0-beta.markdown:72]: Understood - I'm in a similar place and would like to see it so we have an easy answer for the people that keep running into it.

**Review by amorde — APPROVED** [review]

**Pull request merged**